src/core/continuous_space.jl

```diff
@@ -315,19 +315,27 @@ function elastic_collision!(a, b, f = nothing)
   return true
 end
 
+# TODO: Add "true_interacting_pairs" where it is guaranteed that each agent

```

Comment:
We could add a type keyword with [:unique, :single, :all] as options instead of three functions perhaps.

Reply:
This sounds like a good idea to me but I am not sure what `:unique` means. Does it mean the "true" ones?

Reply:
Well, we can deal with semantics later. I just added that since you put "that yields *unique* pairs" in the new doc string.